android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/MapScreen.kt

```diff
@@ -47,6 +47,36 @@ class MapScreen(
             }
             .build()
         )
+        .addAction(
+          Action.Builder()
+            .setIcon(
+              CarIcon.Builder(
+                IconCompat.createWithResource(
+                  carContext,
+                  android.R.drawable.ic_menu_search
+                )
+              ).build()
+            )
+            .setOnClickListener {
+              screenManager.push(SearchScreen(carContext))
+            }
+            .build()
+        )
+        .addAction(
+          Action.Builder()
+            .setIcon(
+              CarIcon.Builder(
+                IconCompat.createWithResource(
+                  carContext,
+                  android.R.drawable.ic_menu_info_details
+                )
+              ).build()
+            )
+            .setOnClickListener {
+              screenManager.push(PaneScreen(carContext))
+            }
+            .build()
+        )
```

Comment:
Is this working for you? I'm running into a crash because of too many actions

```
2022-07-14 14:03:36.044 6836-6836/com.mapbox.maps.testapp.auto E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.mapbox.maps.testapp.auto, PID: 6836
    java.lang.RuntimeException: java.lang.IllegalArgumentException: Action list exceeded max number of 4 actions
        at androidx.car.app.utils.RemoteUtils.lambda$dispatchCallFromHost$0(RemoteUtils.java:155)
        at androidx.car.app.utils.RemoteUtils$$ExternalSyntheticLambda2.run(Unknown Source:6)
        at androidx.car.app.utils.ThreadUtils.runOnMain(ThreadUtils.java:39)
        at androidx.car.app.utils.RemoteUtils.dispatchCallFromHost(RemoteUtils.java:147)
        at androidx.car.app.utils.RemoteUtils.lambda$dispatchCallFromHost$1(RemoteUtils.java:186)
        at androidx.car.app.utils.RemoteUtils$$ExternalSyntheticLambda3.run(Unknown Source:8)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7664)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: java.lang.IllegalArgumentException: Action list exceeded max number of 4 actions
        at androidx.car.app.model.constraints.ActionsConstraints.validateOrThrow(ActionsConstraints.java:198)
        at androidx.car.app.navigation.model.NavigationTemplate$Builder.setMapActionStrip(NavigationTemplate.java:377)
        at com.mapbox.maps.testapp.auto.car.MapScreen.onGetTemplate(MapScreen.kt:62)
        at androidx.car.app.Screen.getLastTemplateInfo(Screen.java:394)
        at androidx.car.app.ScreenManager.getTopTemplate(ScreenManager.java:283)
        at androidx.car.app.AppManager$1$$ExternalSyntheticLambda3.dispatch(Unknown Source:2)
        at androidx.car.app.utils.RemoteUtils.lambda$dispatchCallFromHost$0(RemoteUtils.java:150)
        at androidx.car.app.utils.RemoteUtils$$ExternalSyntheticLambda2.run(Unknown Source:6) 
        at androidx.car.app.utils.ThreadUtils.runOnMain(ThreadUtils.java:39) 
        at androidx.car.app.utils.RemoteUtils.dispatchCallFromHost(RemoteUtils.java:147) 
        at androidx.car.app.utils.RemoteUtils.lambda$dispatchCallFromHost$1(RemoteUtils.java:186) 
        at androidx.car.app.utils.RemoteUtils$$ExternalSyntheticLambda3.run(Unknown Source:8) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7664) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
```